### DISCUSSION TITLE
Upgrade Issue

### DISCUSSION THREAD

**Discussion by Cory74** [netbox-community/netbox#12999] [Help Wanted!] [answered]
Hi 

I am having an issue with my current install of netbox every time I follow the upgrade process at https://github.com/netbox-community/netbox/blob/develop/docs/installation/upgrading.md I get a Bad Gateway Error 502 

Tried both release 3.5.3 and .4
Current Version 3.5.1 
running on a VM using RHEL 8  not a docker image

Any help would be greatly appreciated.

**Comment by abhi1693** [comment]
Could you share your logs?

**Comment by Cory74** [comment]
I think this is the issue 

File "/opt/netbox/venv/lib/python3.10/site-packages/netbox_dns/forms/view.py", line 9, in <module>
    from utilities.forms import TagFilterField
ImportError: cannot import name 'TagFilterField' from 'utilities.forms' (/opt/netbox/netbox/utilities/forms/__init__.py)

Any idea how to resolve it?

**Reply by abhi1693** [reply, marked as answer]
You are probably using an incompatible version of the dns plugin with your NetBox. I'd suggest you look into the newer maintained version at https://github.com/peteeckel/netbox-plugin-dns

**Reply by Cory74** [reply]
I removed the plugin and did the upgrade and it worked fine  

Thanks for your help
